src/elements/content-sidebar/BoxAISidebar.tsx

```diff
@@ -80,6 +81,7 @@ const BoxAISidebar = (props: BoxAISidebarProps) => {
         fileID,
         getSuggestedQuestions,
         isIntelligentQueryMode,
+        isOpen,
```

Comment:
When looking at the corresponding EUA change, I've come to a conclusion, that I've approved this prematurely :)
Do we need to get this prop from outside? I believe we could pass it to `toBoxAISidebarContent` always as `true`, because if the sidebar is closed, `BoxAISidebarContent` component will never be rendered.

Reply:
I agree, let's remove it from props here.

Reply:
Updated